treeherder/webapp/api/bugzilla.py

```diff
@@ -43,6 +43,9 @@ def create_bug(self, request):
             'description': description,
             'comment_tags': "treeherder",
         }
+        if len(params.get("crash_signature") > 2048):
```

Comment:
If `crash_signature` is not set, this will result in:
`TypeError: object of type 'NoneType' has no len()`

I think this check should also be moved up higher (fail early and all that).